src/FamilyHubs.ReferralUi.Ui/Pages/Error/404.cshtml

```diff
@@ -3,19 +3,19 @@
     ViewData["Title"] = "Page not found";
 }
 <a href="/" class="govuk-back-link" data-testid="back-button">Home</a>
-<main class="govuk-main-wrapper " id="main-content" role="main">
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Page not found</h1>
-        <p>
-            If you typed the web address, check it is correct.
-        </p>
-        <p>
-            If you pasted the web address, check you copied the entire address.
-        </p>
-        <p>
-            If the problem continues, email us at <a href="mailto:connect-support-for-your-family.service@education.gov.uk.">connect-support-for-your-family.service@education.gov.uk.</a>
-        </p>
+<main class="govuk-main-wrapper custom-main" id="main-content" role="main">
```

Comment:
Do you have a `main` in the layout? If so, you shouldn't have main in here.

Reply:
@lazcool No. We have main on every single page individually. Was done like that from the beginning.